python/lsst/analysis/tools/actions/vector/selectors.py

```diff
@@ -47,17 +48,22 @@
 from lsst.pex.config.listField import ListField
 
 from ...interfaces import KeyedData, KeyedDataSchema, Vector, VectorAction
-from ...math import divide
+from ...math import divide, fluxToMag
 
 
 class SelectorBase(VectorAction):
     plotLabelKey = Field[str](
         doc="Key to use when populating plot info, ignored if empty string", optional=True, default=""
     )
 
-    def _addValueToPlotInfo(self, value, **kwargs):
-        if "plotInfo" in kwargs and self.plotLabelKey:
-            kwargs["plotInfo"][self.plotLabelKey] = value
+    def _addValueToPlotInfo(self, value, plotLabelKey=None, **kwargs):
+        if "plotInfo" in kwargs:
+            if self.plotLabelKey:
```

Comment:
This seems backwards.  Generally if a keyword is set it should override any `self.` parameters.  So I think the first check should be `if plotLabelKey is not None:`.